tests/test_ext_autodoc.py

```diff
@@ -1750,27 +1758,31 @@ def test_autodoc_TypeVar(app):
         '',
         "   alias of TypeVar('T1')",
         '',
+        '',
         '.. py:data:: T3',
         '   :module: target.typevar',
         '',
         '   T3',
         '',
         "   alias of TypeVar('T3', int, str)",
```

Comment:
I'll make these arguments hyperlinked in the next step.

Reply:
I don't see anything for the `bound`  parameter for TypeVar.  It should be accessible via the `__bound__` attribute.  It's important information to display.

Reply:
Could you file a new issue? Then we'll consider improving it.